device_e2e/aio/stress_results.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+import threading
+import math
+
+
+class Measurements(object):
+    """Object which defines the measurements we take durring a stress test"""
+
+    def __init__(self):
+        self.lock = threading.Lock()
+        self.peak_reconnect_time = 0
+        self.peak_resident_memory_mb = 0
+        self.peak_telemetry_arrival_time = 0
+        self.queued_telemetry_messages = 0
+        self.peak_queued_telemetry_messages = 0
+        self.total_elapsed_time = 0
+
+
+def round(x):
+    """
+    round up if the decimal part of x is >= .5, otherwise round down
+    """
+    return math.floor(x + 0.5)
+
+
+def two_decimal_places(x):
+    """
+    round x to two decimal places
+    """
+    return round(x * 100) / 100
```

Comment:
Why not use the [python built-in `round()` function](https://docs.python.org/3/library/functions.html#round)?

Reply:
Because I only looked in the math module. :)